MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1168 @@
+        {
+            comboBoxItem.MaxDropDownHeight = 400;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Loading complex type property values in combo box editor
+
+You can display the complex type property values in combo box editor by using the DisplayMemberPath and SelectedValuePath properties of DataFormComboBoxItem. You need to use `GenerateDataFormItem `event to set [DisplayMemberPath]`Link` and [SelectedValuePath]`Link` property value of DataFormComboBoxItem for complex type property.
+
+N> Class cannot be directly set as data type for combo box editor in this complex type scenario.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.DataObject = new ContactModel();
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+    dataForm.RegisterEditor("EmployeeDetails", DataFormEditorType.ComboBox);
+ 
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "EmployeeDetails" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {   
+            comboBoxItem.DisplayMemberPath = "ID";
+            comboBoxItem.SelectedValuePath = "Name";
+        }
+    } 
+ 
+    public class DataFormItemsSourceProvider  : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            if (sourceName == "EmployeeDetails")
+            {
+                List<EmployeeInfo> details = new List<EmployeeInfo>();
+                details.Add(new EmployeeInfo() { ID = 1, Name =  "Eric"  });
+                details.Add(new EmployeeInfo() { ID = 2, Name = "James"  });
+                details.Add(new EmployeeInfo() { ID = 3, Name = "Jacob"  });
+                details.Add(new EmployeeInfo() { ID = 4, Name = "Lucas"  });
+                details.Add(new EmployeeInfo() { ID = 5, Name = "Mark"  });
+                details.Add(new EmployeeInfo() { ID = 6, Name = "Aldan"  });
+                details.Add(new EmployeeInfo() { ID = 7, Name = "Aldrin"  });
+                details.Add(new EmployeeInfo() { ID = 8, Name = "Alan"  });
+                details.Add(new EmployeeInfo() { ID = 9, Name = "Aaron"  });
+
+                return details;
+            }
+        return new List<string>();
+        }
+    }
+
+    public class ContactInfo
+    {
+        [Display(Name ="First Name")]
+        public String FirstName { get; set; } 
+        public string EmployeeDetails { get; set; }
+    }
+
+    public class EmployeeInfo
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## AutoComplete editor
+
+In the autocomplete editor, the [SfAutoComplete](https://help.syncfusion.com/maui/autocomplete) is loaded.
+
+### Customizing ItemsSource of autocomplete editor
```

Comment:
### Changing the ItemsSource of autocomplete editor (4#)

Reply:
title changed